src/reactviews/pages/ConnectionDialog/components/advancedOptionsDrawer.component.tsx

```diff
@@ -35,11 +37,37 @@ export const AdvancedOptionsDrawer = ({
     setIsAdvancedDrawerOpen: React.Dispatch<React.SetStateAction<boolean>>;
 }) => {
     const context = useContext(ConnectionDialogContext);
+    const [searchSettingsText, setSearchSettingText] = useState<string>("");
+    const [userOpenedSections, setUserOpenedSections] = useState<string[]>(["General"]);
 
     if (context === undefined) {
         return undefined;
     }
 
+    const doesGroupHaveVisibleOptions = (group: ConnectionComponentGroup) => {
+        return group.options.some((optionName) => {
+            if (context.state.formComponents[optionName]?.hidden === true) {
+                return false;
+            }
```

Comment:
Do we ever hide advanced options?  This entire statement may be a no-op.

Reply:
Don't think so just copied it from existing logic. Will remove it.